landing-page/content/common/hive-quickstart.md

```diff
@@ -0,0 +1,124 @@
+weight: 100
+url: hive-quickstart
+aliases:
+    - "quickstart"
+    - "quickstarts"
+    - "getting-started"
+disableSidebar: true
+disableToc: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<!-- {{% quickstarts %}} -->
+
+## Hive and Iceberg Quickstart
+
+This guide will get you up and running with an Iceberg and Hive environment, including sample code to
+highlight some powerful features. You can learn more about Iceberg's Hive runtime by checking out the [Hive](../docs/latest/hive/) section.
+
+- [Docker Images](#docker-images)
+- [Creating a Table](#creating-a-table)
+- [Writing Data to a Table](#writing-data-to-a-table)
+- [Reading Data from a Table](#reading-data-from-a-table)
+- [Next Steps](#next-steps)
+
+### Docker Images
+
+The fastest way to get started is to use [Apache Hive images](https://hub.docker.com/r/apache/hive) 
+which provides a SQL-like interface to create and query Iceberg tables from your laptop. You need to install the [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+Take a look at the Tags tab in [Apache Hive docker images](https://hub.docker.com/r/apache/hive/tags?page=1&ordering=-last_updated) to see the available Hive versions.
+
+Set the version variable. For example:
+```sh
+export HIVE_VERSION=4.0.0-alpha-2
+```
+
+Start the container, using the option --platform linux/amd64 for a Mac with an M-Series chip:
+```sh
+docker run -d --platform linux/amd64 -p 10000:10000 -p 10002:10002 --env SERVICE_NAME=hiveserver2 --name hive4 apache/hive:${HIVE_VERSION}
+```
+
+The docker run command above configures Hive to use the embedded derby database for Hive Metastore. Hive Metastore functions as the Iceberg catalog to locate Iceberg files, which can be anywhere. 
+
+Give HiveServer (HS2) a little time to come up in the docker container, and then start the Hive Beeline client using the following command to connect with the HS2 containers you already started:
+```sh
+docker exec -it hive4 beeline -u 'jdbc:hive2://localhost:10000/'
+```
+
+The hive prompt appears:
+```sh
+0: jdbc:hive2://localhost:10000>
+```
+
+You can now run SQL queries to create Iceberg tables and query the tables. For example:
```

Comment:
Very minor nit (and opinion): I think this reads better without "For example: " at the end. I saw that once further up as well.

Reply:
SGTM, tks. 
![image](https://github.com/apache/iceberg-docs/assets/16125343/92320c53-1608-4eb9-9769-fea7ff746002)
Got the other one too.